app/src/main/res/layout/fragment_transaction_details.xml

```diff
@@ -151,6 +153,19 @@
                     android:drawablePadding="6dp"
                     android:gravity="start|center_vertical"
                     android:text="@string/tx_details_view_on_etherscan" />
+
+                <TextView
```

Comment:
The confirm button should be stick to the bottom tab bar and not scroll (see ticket description)

Reply:
there is no bottom tab bar in the transaction details.

Reply:
Unless we have to change the navigation, but what I remember from planning is that you mentioned you also believed the mocks were wrong.

Reply:
Here it could mean that confirm button is not scrollable and is always at the bottom. But please clarify it with UX